waspc/packages/deploy/src/providers/fly/CommonOptions.ts

```diff
@@ -13,3 +14,8 @@ export interface DbOptions {
 export interface LocalBuildOptions {
 	buildLocally: boolean;
 }
+
+export interface SecretsOptions {
+	serverSecret: string[];
```

Comment:
Maybe go with plural `serverSecrets`, is a `string` the best way to represent a list of key values?
```ts
type Secret { key: string; value: string }
```

Reply:
The interface names are a convention of commander.js. Our option name is `--server-secret` which they will put into a variable `serverSecret`.